crates/astria-sequencer/src/authority/storage/keys.rs

```diff
@@ -1,24 +1,58 @@
+use crate::accounts::AddressBytes;
+
 pub(in crate::authority) const SUDO: &str = "authority/sudo";
-pub(in crate::authority) const VALIDATOR_SET: &str = "authority/validator_set";
+pub(in crate::authority) const VALIDATOR_PREFIX: &str = "authority/validator/";
+pub(in crate::authority) const VALIDATOR_COUNT: &str = "authority/validator_count";
 pub(in crate::authority) const VALIDATOR_UPDATES: &str = "authority/validator_updates";
 
+pub(in crate::authority) const PRE_ASPEN_VALIDATOR_SET: &str = "authority/validator_set"; // Deprecated post Aspen upgrade
+
+pub(in crate::authority) fn validator<TAddress: AddressBytes>(address: &TAddress) -> String {
+    format!(
+        "{}/{}",
```

Comment:
The prefix already has a trailing `/` (as per other prefixes), so we don't want one in the format string here:
```suggestion
        "{}{}",
```

Reply:
Good catch! Done in e9877a8.